lib/ioki/apis/operator_api.rb

```diff
@@ -395,6 +395,11 @@ class OperatorApi
         :blacklisted_travel_combination,
         base_path:   [API_BASE_PATH, 'products', :id],
         model_class: Ioki::Model::Operator::BlacklistedTravelCombination
+      ),
+      Endpoints.crud_endpoints(
```

Comment:
If we can't `update` it (I assume?), should we include `except: [:update]`?

Reply:
Totally missed that one! Thanks for pointing it out. I will exclude it 👍

Reply:
Done!